torch/_inductor/lowering.py

```diff
@@ -255,7 +255,7 @@ def _register_lowering(
 
     @functools.wraps(decomp_fn)
     def wrapped(*args, **kwargs):
-        args = list(args)
+        args: Union[List[Any], Tuple[Any, ...]] = list(args)
```

Comment:
im pretty sure *args is always wrapped as a tuple.. could just do `Tuple[Any, ...]]` maybe.

Reply:
I've tried getting rid of the `list(args)` there in the first version, but that has led to some cpp wrapper test failures: https://hud.pytorch.org/pytorch/pytorch/commit/ed026d4019774cd72e5f3c45d25f55a90d3c5943 . So opted for keeping the `list(args)` there, but because of this a few lines below:

```
        if len(args) == 1 and isinstance(args[0], (list, tuple)):
            unpacked = True
            args = args[0]
```

had to keep the `Union` of list and tuple.